docs/eventlisteners.md

```diff
@@ -78,28 +156,28 @@ Event Interceptors can take several different forms today:
 
 ### Webhook Interceptors
 
-Webhook interceptors allow users to configure an external k8s object which
+Webhook Interceptors allow users to configure an external k8s object which
 contains business logic. These are currently specified under the `Webhook`
 field, which contains an
 [`ObjectReference`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
-to a Kubernetes Service. If a Webhook interceptor is specified, the
+to a Kubernetes Service. If a Webhook Interceptor is specified, the
 `EventListener` sink will forward incoming events to the service referenced by
-the interceptor over HTTP. The service is expected to process the event and
+the Interceptor over HTTP. The service is expected to process the event and
 return a response back. The status code of the response determines if the
-processing is successful - a 200 response means the interceptor was successful
-and that processing should continue, any other status code will halt trigger
-processing. The returned body is used as the new event payload by the
-EventListener and passed on the `TriggerBinding`. An interceptor has an optional
-header field with key-value pairs that will be merged with event headers before
-being sent;
+processing is successful - a 200 response means the Interceptor was successful
+and that processing should continue, any other status code will halt Trigger
+processing. The returned request (body and headers) is used as the new event
+payload by the EventListener and passed on the `TriggerBinding`. An Interceptor
+has an optional header field with key-value pairs that will be merged with event
+headers before being sent;
 [canonical](https://github.com/golang/go/blob/master/src/net/http/header.go#L214)
```

Comment:
I think that the line number for this link has changed. Maybe we can link to this instead in-case the line number changes again? https://golang.org/pkg/net/textproto/#CanonicalMIMEHeaderKey

Reply:
done